### PR TITLE
styleguide - headings and links styling changes

### PR DESCRIPTION
**Pull request by katlazarova**

### CODE REVIEW

File: web/themes/custom/pineapple/sass/pineapple.styles.scss [hunk 1]
```diff
@@ -1,3 +1,7 @@
+// Import external libraries. Grunt-sass has been configured to have the
+// theme's npm node_modules and Bower libraries directory as an include path.
+@import "breakpoint-sass/stylesheets/breakpoint";
+
 @charset "UTF-8";
```
Comment (ElXyfer) [inline web/themes/custom/pineapple/sass/pineapple.styles.scss]: This needs to go to the top of the file. There's a php storm warning.

File: web/themes/custom/pineapple/sass/core/01_variables/_typography.scss [hunk 2]
```diff
@@ -4,19 +4,31 @@
...
 /// The base font family for the site.
-$base-font-family: 'Montserrat', sans-serif;
+$base-font-family: 'Lato', sans-serif;
+@font-face {
+  font-family: ‘Lato’;
+  src: url('../../../fonts/Lato/Lato-Regular.ttf') format('truetype');
+  font-weight: normal;
+  font-style: normal;
+}
+
```
Comment (ElXyfer) [inline web/themes/custom/pineapple/sass/core/01_variables/_typography.scss]: You're only adding one font but the fonts folder has loads of fonts, either add some more here or remove some from the fonts folder

File: web/themes/custom/pineapple/sass/pineapple.styles.scss (hunk 1 above)
Comment (katlazarova) [reply web/themes/custom/pineapple/sass/pineapple.styles.scss]: done

File: web/themes/custom/pineapple/sass/core/01_variables/_typography.scss (hunk 2 above)
Comment (katlazarova) [reply web/themes/custom/pineapple/sass/core/01_variables/_typography.scss]: done

File: drushrc.php [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<?php
+
+$options['uri'] = 'http://localhost:8888';
```
Comment (ElXyfer) [inline drushrc.php]: For some reason the drush rc file isn't doing what it should, You can use the drush.yml file in the drush folder in Your project root instead and You can have the following inside it:

`options:
  uri: 'http://localhost:8888'`

once You've done that verify it works and delete this file.

File: web/themes/custom/pineapple/sass/base/typography/_links.scss [hunk 4]
```diff
@@ -0,0 +1,37 @@
+.links {
```
Comment (ElXyfer) [inline web/themes/custom/pineapple/sass/base/typography/_links.scss]: Remove this links class.

File: web/themes/custom/pineapple/sass/base/typography/_links.scss [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+  a {
+    text-decoration: none;
+    color: $light-grey;
+
+    &:hover {
+      color: $dark-grey;
+      cursor: pointer;
+    }
+  }
+}
```
Comment (ElXyfer) [inline web/themes/custom/pineapple/sass/base/typography/_links.scss]: Delete this section.

Comment (katlazarova) [reply web/themes/custom/pineapple/sass/base/typography/_links.scss]: done

File: web/themes/custom/pineapple/sass/base/typography/_links.scss (hunk 4 above)
Comment (katlazarova) [reply web/themes/custom/pineapple/sass/base/typography/_links.scss]: done.

File: drushrc.php (hunk 3 above)
Comment (katlazarova) [reply drushrc.php]: done.

**Pull request merged**